src/QMCWaveFunctions/WaveFunctionComponent.h

```diff
@@ -532,6 +541,12 @@ class WaveFunctionComponent : public QMCTraits
                                    std::vector<ValueType>& ratios,
                                    Matrix<ValueType>& dratios);
 
+  
+  virtual void detRatios(Vector<ValueType>& ratios)
```

Comment:
Please remove from WFC and keep it only to MSD.

Reply:
What does this function calculate?

Reply:
Removed from WFC, functionality only at MSD level now.

Inline documentation added.